### ISSUE TITLE
Add more default search filters to schedules list

### ISSUE THREAD

**Issue by dsesami** [ansible/awx#6519]
##### ISSUE TYPE
 - Feature-ish bug

##### SUMMARY
Follow up to #5857 
@mabashian:
> Yea there probably are other fields that a user might want to search on. Maybe enabled/disabled, searching on the various dates (created, edited, start datetime, etc). It may also be helpful to give them a quick filter to show schedules that will never run again (end datetime is < now).

@jlmitch5:
> +1 those all sound like great search keys @mabashian
> the date ones may? be blocked by #6214 currently the search does not implement the date input element (but could be made as an enhancement). that maybe should be a follow-up issue? up to you

This should apply to schedule lists for JTs, WFJTs, Projects, and the overall schedules page.

**Comment by dsesami** [comment]
This is unimplemented as of Sep 15.

**Comment by wenottingham** [comment]
Implemented via advanced search, moving to needs_test

**Comment by unlikelyzero** [comment]
@wenottingham we definitely cannot search for datetime in advanced search. I'll leave this in needs_test so that @akus062381 can determine what we can and cannot query on the /schedules/ page and JT/schedule page

Schedules Page
- Maybe enabled/disabled searches
- searching on the various dates (created, edited, start datetime, etc). 
- It may also be helpful to give them a quick filter to show schedules that will never run again (end datetime is < now).

JT Schedules Searchterms
- Maybe enabled/disabled searches
- searching on the various dates (created, edited, start datetime, etc). 
- It may also be helpful to give them a quick filter to show schedules that will never run again (end datetime is < now).

Note: datetimes are going to be localized soon and we'll need to be able to search for all datetime formats we display on the schedules page

If these searches do not work, we'll file RFEs for future search support

**Comment by akus062381** [comment]
FINDINGS:

- if I set the key to modified (as in date) and I choose exact as the lookup type
           and I grab the value from the API- for example, 2021-03-08T15:14:22.621151Z
           it returns that specific value to me.
           BUT
           if I leave the key as modified and change the Lookup type to icontains
           and enter search value 2021-03-08, it finds nothing
           ** This is true for all the date field searches- only lookup type Exact works, with the exact value entered from the API,
             which I doubt a user would know.
- I have a tough time understanding the purpose of the extra_data search. I don't see a field anywhere in the UI called Extra 
            Data, and although the search * kind of * works, a user would never know what to enter (because they'd have to add 
            a precise value for it to work) in order to return the information they were looking for.
- Searching for an inventory connected to a certain schedule in the API returns:
![image](https://user-images.githubusercontent.com/63685497/110647678-08e8b000-8186-11eb-9e93-140240d8d22d.png)

- When searching for a unified job template, using values I found in the API, the search didn't work. Tried multiple times.
     
![image](https://user-images.githubusercontent.com/63685497/110648267-8f9d8d00-8186-11eb-8545-ed114e583161.png)

- Regarding the search for Enabled: would a user know it's a boolean field and type true or false? I wouldn't have done that without checking the API.
- Generally speaking, searching for dates- whether it's start date, end date, etc- doesn't work unless you enter the EXACT value shown in the API, which again, I don't believe most users would know when searching for something.... especially since seconds are included.
- The credential search doesn't appear to be functional and doesn't make sense to me, as I see no way to associate a credential with a schedule.

**Comment by mabashian** [comment]
Things that we should look at adding to the basic search dropdown:

Enabled

**Comment by unlikelyzero** [comment]
@wenottingham @mabashian moving this to backlog as it's an enhancement
